### ISSUE TITLE
Stop changing order in the output

### ISSUE THREAD

**Issue by ibobak** [Patric-R/jvmtop#110]
Could you please stop changing order of the java processes which you're outputting?
Because it is hard to read...

**Comment by kamstrup** [comment]
I guess that's the whole idea of top-like programs. That they sort by cpu usage with live updates?

But maybe an option to sort by PID or something; which should give a stable view.

**Comment by patric-r** [comment]
I agree with @kamstrup, it's part of the "top" concept and matches the behavior of other/similar applications.
So it's not a bug but a feature.

Sorting by other criteria is legitimate feature request, however.
